R/scopus_mods.R

```diff
@@ -132,13 +149,15 @@ scopus_mods_params <- function(scopus, sid, kthid_orcid_lookup = kthid_orcid())
     select(-auid) |>
     tibble::rowid_to_column()
 
+  # TODO: for many scopusids, get the raw_org... find all with afid for KTH
+  # run the re_kth - does it match for all those records.
   re_kth <- paste0(
-    "kth|KTH|roy.*?inst.*?tech.*?|Roy\\. Inst\\. T|alfven|",
-    "kung.*?tek.*?hog|kgl.*?tek.*?|kung.*?tek.*?hg.*?|roy.*?tech.*?univ.*?"
+    "kth|roy.*?inst.*?tech.*?|roy\\. inst\\. t|alfven|",
+    "kung.*?tek.*?h[o\u00f6]g.*?|kgl.*?tek.*?|kung.*?tek.*?hg.*?|roy.*?tech.*?univ.*?"
```

Comment:
fix r cmd check warning about using non-ascii characters in R code (used stringi to escape "ö").